### DISCUSSION TITLE
WLAN Probleme auf einer 6591 Providerbox ("kdg" branding)

### DISCUSSION THREAD

**Discussion by gh47110815** [Freetz-NG/freetz-ng#592] [Allgemein]
Hallo,

**zur Umgebung:**
ich bin neu hier und habe Freetz-NG (seit ca. 1 Woche) auf einer 6591 Providerbox mit "lgi" Branding laufen und **bisher** keine Probleme (**edit**: erst beim Tippen dieser Zeilen ist mir aufgefallen, dass das natürlich eine gewagte Aussage sein könnte, denn an dieser "funktionierenden" Box im Keller fiele ein Problem - z.B. WLAN - auch nicht gleich auf. **edit2**: ein ganzer Nachmittag als WLAN-Client bestätigt: das WLAN dieser 6591 läuft problemlos!).
Wobei WICHTIG zu erwähnen ist, dass diese Box als Mesh-Client und im IP-Client Mode läuft, da am "echten" Kabelanschluss eine unberührte 6591 von Vodafone läuft. An diese lange ich auch nicht ran ;-) Andere FRITZ!Boxen im LAN dienen rein dem "Luxus" höherwertige WLAN-Repeater zu haben, die dann eben zusätzlich noch DVB-C TV anbieten. Daher dieses eher ungewöhnliche Setup.
Auch weiss ich NIE, was ich von Vodafone zu erwarten habe oder ob ich diesen Vertrag mal kündige. Daher betreibe ich deren 6591 (also den echten Internet-Router) auch nicht als Mesh-Master - sondern als Mesh-Client - um diese evtl. schnell und einfach tauschen zu können.

Den Mesh-Master spielt eine andere 6591 mit "kdg" Branding, die ebenfalls im IP-Client-Mode läuft. Diese läuft bisher einwandfrei unter ffritz (gepachted sowohl mit dem aktuell verfügbaren Release 07.29 als auch mit der  neuesten Laborversion 07.39 - 97972). Sie steht auch an der meist frequentierten Stelle im WLAN, versorgt also die meisten WLAN Clients hier bislang problemlos mit WLAN 

**... nun zum Problem ...**
Das freetz-ng image der 6591 mit "lgi" Branding (Mesh-Client) habe ich auch mal auf die 6591 mit "kdg" Branding (Mesh-Master) gespielt und aktiviert. Sieht eigentlich auf den ersten Blick ganz gut aus - die 6591 Mesh-Master bootet, avm- und freetz-NG GUIs sind ansprechbar .... DEM ANSCHEIN NACH ist alles OK - aber die WLAN-Clients können sich nicht mehr mit dem WLAN (dieser Box) verbinden.

**... und zu meiner Frage ...**
Was meint Ihr an was das liegen kann? Welche Logfiles wären da interessant um das Problem einzugrenzen und ggf. zu lösen?
Ich selbst bin kein Entwickler, nur Anwender - könnte aber natürlich bei Tests und der Problemeingrenzung helfen. Bin auch ganz neu (seit 1 Woche) hier.

Daher habe ich mal diese Diskussion eröffnet - hoffe das ist ok.

**Comment by fda77** [comment]
Gehts ohne Freetz? Was heist "nicht verbinden"? Ist Wlan bei Clients an? Auf der Box? Ne SSID zu sehne? Geht die Verbindubg selbst? Kein DHCP? Kein Routing? ....

**Reply by gh47110815** [reply]
a) Ja - ohne Freetz gehts. Also Bootsbank-Switch zurück auf ffritz und alles ist ok. 
b) SSID ist zu sehen. Die Clients können sich nicht verbinden. Bei iOS Clients kommt der normale Dialog zum Verbinden mit dem WLAN <SSID>, aber beim Verbinden ein Fehler und man muss das Password erneut eingeben. Aber auch das Richtige PW bringt nichts. Habe auch schon SSID und Password geändert (über die avm GUI der 6591) - Clients können sich am Mesh-Master einfach nicht verbinden. An anderen Mesh-Clients klappt es aber. Wirklich seltsam.

Zu DHCP oder Routing kann man ohne Verbindung natürlich noch gar nichts sagen.

**Reply by gh47110815** [reply]
Update: Heute treten diese WLAN-Probleme auch an der anderen 6591 (Mesh-Client) auf, wenn sich WLAN-Clients in deren Umgebung am WLAN anmelden wollen. Sprich: Es muss doch an diesem Freetz-NG-image liegen. Da aktuell der Mesh-Master wieder unter ffritz läuft und die zweite 6591 (Mesh-Client) unter Freetz-NG. 
Das Problem ähnelt dem u.a. [hier](https://www.ip-phone-forum.de/threads/fritz-box-mesh-dhcp-probleme-i-v-m-switches.308814/post-2419493) beschriebenen DHCP Problem, welches sich wie von selbst durch einen Update der avm Firmware in Luft aufgelöst hatte. Daher die rhetorische Frage: Kann sich eventuell in den Freetz-NG Quellen für die 6591 ggf. noch alter Code von avm (vor diesem wundersamen Update) befinden?  

Also noch einmal zusammenfassend:
a) Wenn ich beide 6591 (Mesh-Master und Mesh-Client) mit ffritz Boote gibt es diese WLAN Probleme definitiv nicht.
b) Wenn eine der beiden aber mit Freetz-NG läuft, gibt es WLAN Probleme bei der Verbindung von Clients. Am Mesh-Master quasi immer, am Mesh-Client scheinbar nur sporadisch.

**Comment by fda77** [comment]
Keine Ahnung. Du könntest dir mir tcpdump/wireshare ansehen weshalb die Clients getrennt werden.
Dass der komplette Netzwerkstack bei AVM nicht 100% rund läuft ist schon lange bekannt

**Reply by gh47110815** [reply]
Ja - lass gut sein. Da ich auf Freetz-NG nicht wirklich angewiesen bin - werde ich einfach wieder auf ffritz wechseln und nur sporadisch und wenn man Images für neue avm-Release- oder avm-Labor-Versionen bauen kann dann auf's neue probieren und für Tests zur Verfügung stehen. Have a nice weekend.

**Comment by fda77** [comment]
Jo, wäre mir auch zu stressig. Das kann jemand anderes tun
